fp.js

```diff
@@ -1,34 +1,57 @@
 // Convert the following code to use `.map`
 
 var peoplesAges = []
-var people = [
-  {name: "Jane", age: 32},
-  {name: "Jessica", age: 34},
-  {name: "Jasmine", age: 39}
+var people = [{
+    name: "Jane",
+    age: 32
+  },
+  {
+    name: "Jessica",
+    age: 34
+  },
+  {
+    name: "Jasmine",
+    age: 39
+  }
 ]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
-});
 
+var map = people.map(function(item) {
+  return item.age
+});
 // Convert the following code to use `.filter`
 
 var peopleYoungerThan35 = []
-var people = [
-  {name: "Jane", age: 32},
-  {name: "Jessica", age: 34},
-  {name: "Jasmine", age: 39}
-]
-people.forEach(function(person) {
-  if(person.age < 35){
-    peopleYoungerThan35.push(person)
+var people = [{
+    name: "Jane",
+    age: 32
+  },
+  {
+    name: "Jessica",
+    age: 34
+  },
+  {
+    name: "Jasmine",
+    age: 39
   }
+]
+var old = people.filter(function(person) {
+  return person.age < 35;
 });
-
 // Convert the following to use `.forEach`
 
-var people = [
-  {name: "Jane", age: 32},
-  {name: "Jessica", age: 34},
-  {name: "Jasmine", age: 39}
+var people = [{
+    name: "Jane",
+    age: 32
+  },
+  {
+    name: "Jessica",
+    age: 34
+  },
+  {
+    name: "Jasmine",
+    age: 39
+  }
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(function(person){return person.name.match("s")});
+var nameS = people.forEach(function(person) {
+  return person.name.match("s");
+});
```

Comment:
this is pushing the result of match. Instead want to use the result of match to determine where or not to push the person
```js
const peopleWhoseNamesHaveTheLetterS = []
people.forEach(person => {
  if (person.name.match(/s/i)){
    peopleWhoseNamesHaveTheLetterS.push(person)
  }
})
```